packages/main/src/plugin/kubernetes-client.ts

```diff
@@ -112,6 +112,8 @@ function toPodInfo(pod: V1Pod, contextName?: string): PodInfo {
     engineId: contextName ?? 'kubernetes',
     engineName: 'Kubernetes',
     kind: 'kubernetes',
+    node: pod.spec?.nodeName,
+    namespace: pod.metadata?.namespace,
```

Comment:
As you said above there is already the Namespace field, so why not reusing it? It has the same content. -> https://github.com/containers/podman-desktop/pull/7684/files#diff-8799a75d3c2928c9fe5136fceab66ed0edad4de3a675f9f5f6d3bc3c61f3dd7bL109

Reply:
Maybe you didn't push your commit